static/retirement/v0api/index.md

```diff
@@ -0,0 +1,225 @@
+
+`https://api.beta.ons.gov.uk/v1/data?uri=/economy/nationalaccounts/uksectoraccounts/timeseries/mm23/capstk`
+
+These uris are currently evergreen and so won't change between editions.
+
+### /timeseries
+
+| Version | URI                                                              |
+|---------|------------------------------------------------------------------|
+| Old     | `{domain}/timeseries`                                            |
+| New     | `https://api.beta.ons.gov.uk/v1/search?&content_type=timeseries` |
+
+Parameters:
+
+| Old parameter | New parameter |
+|---------------|---------------|
+| start         | offset        |
+| limit         | limit         |
+
+You will need to add a new parameter to the request to just return the `dataset` content type:
+
+`content_type=dataset`
+
+See the [search response mapping](#search-response-mapping) for the overall response mapping and [search item data mapping](#search-item-data-mapping) for the mappings for individual items.
+
+### /timeseries/{timeseries\_id}
+
+| Version | URI                                           |
+|---------|-----------------------------------------------|
+| Old     | `{domain}/timeseries/{timeseries_id}` |
+| New     | Deprecated                                    |
+
+There is not a direct replacement for this URI but the URIs for timeseries pages can be derived from search:
+
+`https://api.beta.ons.gov.uk/v1/search?q={timeseries_id}&content_type=timeseries`
+
+Using the `uri` parameter of the `item` that matches `timeseries_id` to `cdid` you can then submit that to our data endpoint, for example:
+
+`https://api.beta.ons.gov.uk/v1/data?uri=/economy/nationalaccounts/uksectoraccounts/timeseries/mm23/capstk`
+
+These uris are currently evergreen and so won't change between releases.
+
+### /timeseries/{timeseries\_id}/dataset
+
+| Version | URI                                           |
+|---------|-----------------------------------------------|
+| Old     | `{domain}/timeseries/{timeseries_id}/dataset` |
+| New     | Deprecated                                    |
+
+This endpoint currently returns a 404 for all `timeseries_id` provided and so can already be considered deprecated.
+
+To get a result to what might be expected for this endpoint, you can go to:
+
+`https://api.beta.ons.gov.uk/v1/search?q={timeseries_id}&content_type=timeseries`
+
+See the [search response mapping](#search-response-mapping) for the overall response mapping and [search item data mapping](#search-item-data-mapping) for the mappings for individual items.
+
+### /search
+
+| Version | URI                                                                     |
+|---------|-------------------------------------------------------------------------|
+| Old     | `https://api.beta.ons.gov.uk/v1/search?content_type=timeseries,dataset` |
```

Comment:
this looks like a New endpoint to me - or I'm rather confused - needs some explanation, please